src/Lexer.js

```diff
@@ -28,8 +28,9 @@ import type Settings from "./Settings";
  * - does not match bare surrogate code units
  * - matches any BMP character except for those just described
  * - matches any valid Unicode surrogate pair
- * - matches a backslash followed by one or more letters
- * - matches a backslash followed by any BMP character, including newline
+ * - matches a backslash followed by one or more whitespace characters
+ * - matches a backslash followed by one or more letters then whitespace
+ * - matches a backslash followed by any BMP character
  * Just because the Lexer matches something doesn't mean it's valid input:
```

Comment:
These comment changes are good, but line 23 is still incorrect.

Reply:
I think it's correct.  At least, I don't think `\ ` is a whitespace character; it's a command whose body is space, now covered by line 31.  Anyway, these comments are about the character stream, and "backslash followed by whitespace" (line 31) is different from "whitespace" (line 23).

Reply:
> "backslash followed by whitespace" (line 31) is different from "whitespace"

Point taken.

Sometimes a complicated RegEx pattern comes with comments that lay out in detail the capturing groups. As an example:

```
// Capturing groups:
//    [1] sign
//    [2] hexadecimal integer
//    [3] integer part
//    [4] numerator of a mixed fraction
//    [5] denominator of a mixed fraction
//    [6] decimal fraction of significand, not including decimal point
//    [7] exponent of a number in scientific notation
//    [8] percentage sign
```

The original comments for this RegEx included some of the same ordering, written less formally. I think that has value.

The pattern that you have introduced is in capturing group 2. Could you humor me and move the corresponding comments to be positioned just after line 23?

Reply:
Documenting the capturing groups seems like a great idea!  I've added that.

I hesitate a little to re-organize the top list because it follows a somewhat logical order of what we *want* to match.  Hopefully documenting the capturing groups makes the order we actually match things clearer?

I'm working on a second PR that will hopefully clean up the regexp order, but it requires a different way of handling `\operatorname` so I think it's best left for a separate PR.